### DISCUSSION TITLE
Is it possible to set a specific folder for the cloudinary storage adapter?

### DISCUSSION THREAD

**Discussion by galacticgibbon** [directus/directus#18124] [Archive]
I tried using root (with a folder called directus in the cloudinary account)

STORAGE_LOCATIONS=cloudinary
STORAGE_CLOUDINARY_DRIVER=cloudinary
STORAGE_CLOUDINARY_ROOT=/directus
STORAGE_CLOUDINARY_CLOUD_NAME=*
STORAGE_CLOUDINARY_API_KEY=*
STORAGE_CLOUDINARY_API_SECRET=*
STORAGE_CLOUDINARY_ACCESS_MODE=public

But I get an error when uploading the image:

`[INTERNAL_SERVER_ERROR] No stream returned for file "18ab05e9-efd8-4cfc-9e99-6d154236a79e.png"`

![Screenshot 2023-04-11 at 11 30 35](https://user-images.githubusercontent.com/9589803/231056569-ff20abe4-f82e-472b-9fc7-199396d30d8f.png)

**Comment by rijkvanzanten** [comment]
That sounds like a bug to me! Mind opening [a new issue](https://github.com/directus/directus/issues/new?assignees=&labels=&template=bug_report.yml)? Ty!
